### ISSUE TITLE
Adding keymap descriptions

### ISSUE THREAD

**Issue by tummetott** [RRethy/nvim-treesitter-textsubjects#28]
Hello, 

is there a way to add keymap descriptions? I can add keymap descriptions for textobjects like this:
```lua
textobjects = {
    select = {
        keymaps = {
            ['af'] = { query = '@function.outer', desc = 'outer function' },
            ['if'] = { query = '@function.inner', desc = 'inner function' },
        },
    },
    move = {
        goto_next_start = {
            [']m'] = { query = '@function.outer', desc = 'next method start' },
        },
        goto_next_end = {
            [']M'] = { query = '@function.outer', desc = 'next method end' },
        },
    },
}
```
Is there also a way for textsubjects?

**Comment by RRethy** [comment]
Not possible currently, I'm open to PRs if you want it in a timely manner, otherwise I'll get to it eventually.

**Comment by RRethy** [comment]
Closed by #40
